Sources/PointFree/Account/PrivateRss.swift

```diff
@@ -266,7 +266,7 @@ private func item(forUser user: User, episode: Episode) -> RssItem {
       type: "video/mp4",
       url: episode.fullVideo.downloadUrl
     ),
-    guid: url(to: .episode(.left(episode.slug))),
+    guid: url(to: .episode(.show(.left(episode.slug)))),
```

Comment:
I bundled up the show and index episode routes into a route enum of its own, so that will cause a bit of churn where we link to episodes.

Reply:
I also think we could prob get rid of the `Either<String, Episode.Id>` and just do a `String`. I do think think we ever take advantage of the fact that you can link to an episode with just the id...